server/graphql/v1/queries.js

```diff
@@ -327,14 +343,57 @@ const queries = {
       if (!collective) throw new Error('This collective does not exist');
 
       // Load transactions
-      return collective.getTransactions({
+      const allTransactions = await collective.getTransactions({
+        attributes: attributes,
         order: [['createdAt', 'DESC']],
         type: args.type,
         limit: args.limit,
         offset: args.offset,
         startDate: args.dateFrom,
         endDate: args.dateTo,
       });
+      if (!fetchDataFromLedger) return allTransactions;
```

Comment:
Why execute and wait for `allTransactions` if `fetchDataFromLedger` is true?

Reply:
Humm I get your point, perhaps something fails when fetching data from the ledger and then makes it useless. you're right, I'll fix that.